_drafts/2016-03-22-software-review.Rmd

```diff
@@ -0,0 +1,218 @@
+---
+name: software-review
+layout: post
+title: "Onboarding at ROpenSci: A Year in Reviews"
+date: 2016-03-22
+authors:
+  - name: Noam Ross
+    url: http://twitter.com/noamross
+  - name: Carl Boettiger
+  - name: Jenny Bryan
+  - name: Scott Chamberlain
+  - name: Rich FitzJohn
+  - name: Karthik Ram
+tags:
+  - software
+  - review
+  - infrastructure
+---
+
+Code review, in which peers manually inspect the source code of software
+written by others, is widely recognized as one of the best tools for finding
+bugs in software. Code review is relatively uncommon in *scientific* software
+development, though. Scientists, despite being familiar with the process of
+peer review, often have little exposure code review due to lack of training and
+historically little incentive to share the source code from their research. So
+scientific code, from one-off scripts to reusable R packages, is rarely subject
+to review. Most R packages are subject only to the automated checks required by
+CRAN, which primarily ensure that packages can be installed on multiple systems.
+As such, The burden is on software users to discern well-written and efficient
+packages from poorly written ones.
+
+ROpenSci is a community of developer-scientists, creating R packages for other
+scientists, and our package contributors have a mix of backgrounds. We aim to
+serve our users with high-quality software, but also promote best practices
+among our author base and in the scientific community in general. So for
+the past year, ROpenSci has been piloting a system of peer code review for
+submissions to [our suite of R packages](https://ropensci.org/packages/). Here
+we'll outline how our system works, and what we've learned from our authors and
+reviewers.
+
+# Our System
+
+ROpenSci's package review process owes much to the experiments of others
+(such as the [Mozilla Science Lab](https://mozillascience.org/code-review-
+for-science-what-we-learned)), as well as the [active feedback](https://
+discuss.ropensci.org/t/code-review-onboarding-milestones/180) [from our
+community](https://discuss.ropensci.org/t/how-could-the-onboarding-package-
+review-process-be-even-better/302). Here's how it works: When an author submits
+a package, our editors evaluate it for fit according to our [criteria](https://
+github.com/ropensci/policies#package-fit), then assign reviewers who evaluate
+the package for usability, quality, and style based on our [guidelines](https://
+github.com/ropensci/packaging_guide#ropensci- packaging-guide). After the
```

Comment:
link has a space in it , shoudl be https://github.com/ropensci/packaging_guide#ropensci-packaging-guide